server/lib/nl/detection/utils.py

```diff
@@ -207,3 +209,17 @@ def get_multi_sv(main_vars: List[str], cmp_vars: List[str],
   if not dvars.deduplicate_svs(res.candidates[0]):
     return None
   return res
+
+
+# Gets the query string to use for sv detection.
+def get_sv_detection_query(query: str,
```

Comment:
nit: since we're only doing date string trigger removals, would it be more appropriate for this function to named something like ``remove_date_detection_from_query`` ?

Reply:
The thought behind making this function more general was that in the future we might want to remove other trigger strings. But since we're only removing date for now, I've updated the function to be specific to removing date & we can always make the function more general again in the future when we want to remove more things.